docker-compose-openelis.yml

```diff
@@ -0,0 +1,229 @@
+version: '3.6'
+
+x-logging:
+  &local-logging
+  driver: "local"
+  options:
+    max-size: "20m"
+    max-file: "50"
+
+x-loki-logging:
+  &loki-logging
+  driver: "loki"
+  options:
+    max-size: "20m"
+    max-file: "50"
+    loki-url: "[% loki_url %]"
+    loki-external-labels: "container_name={{.Name}},hostname=[% host_name %]"
+    loki-pipeline-stages: |
+      - multiline:
+          firstline: '^\d{4}-\d{2}-\d{2}[T ]?\d{2}:\d{2}:\d{2}.\d{3}'
+      - labeldrop:
+          - compose_project
+          - compose_service
+          - filename
+          - source
+          - stream
+          - swarm_service
+          - swarm_stack
+services:
+  certs:
+    container_name: oe-certs
+    image: itechuw/certgen:main
```

Comment:
I see inconsistencies with what suppose to be `latest` docker image (`main`, `develop`), can we advice them to push docker images with the latest tag for bleeding edge